salt/metalk8s/defaults.yaml

```diff
@@ -61,10 +61,20 @@ ca:
   signing_policy:
     days_valid: 365
 
+front_proxy_ca:
```

Comment:
Sorry for that but suggest to have ca just bellow in `front_proxy` add a section `ca`